tests/pauli/test_pauli_arithmetic.py

```diff
@@ -565,14 +565,6 @@ def test_hamiltonian_wire_order(self):
 
         assert qml.equal(op, id)
 
-    def test_hamiltonian_complex_coeffs(self):
```

Comment:
Why this test is removed?

Reply:
It is deleted in both the base branch and here. Addressing an old edge case I fixed.